README.md

```diff
@@ -69,7 +69,8 @@ gulp.task('sass', function () {
 `gulp-sass` can be used in tandem with [gulp-sourcemaps](https://github.com/floridoo/gulp-sourcemaps) to generate source maps for the Sass to CSS compilation. You will need to initialize [gulp-sourcemaps](https://github.com/floridoo/gulp-sourcemaps) prior to running `gulp-sass` and write the source maps after.
 
 ```javascript
-var sourcemaps = require('gulp-sourcemaps');
+'use strict';
```

Comment:
Not necessary for ES6 modules: https://stackoverflow.com/a/31685340

Reply:
You need to do `use strict` for node 4 in order to enable ECMA6 features